core/src/main/java/com/amplifyframework/predictions/options/TextToSpeechOptions.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.options;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.amplifyframework.core.async.Options;
+
+/**
+ * Options for text to speech conversion operation.
+ */
+public final class TextToSpeechOptions implements Options {
+    private final String voiceType;
+
+    private TextToSpeechOptions(final Builder builder) {
+        this.voiceType = builder.getVoiceType();
+    }
+
+    /**
+     * Gets the custom voice type if specified.
+     * Null otherwise.
+     * @return the custom voice type
+     */
+    @Nullable
+    public String getVoiceType() {
```

Comment:
This is `String` since `VoiceType` is in the plugin, right? Any other options? (I agree, this is better than including the current `VoiceType` into core!)

Reply:
I figured it out! `VoiceType` is now a core predictions interface with `String getVoice();` method, and `AWSVoiceType` implements that voice type inside the plugin!